src/Aspire.Hosting.Docker/DockerComposePublisherLoggerExtensions.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Aspire.Hosting.Docker;
+
+/// <summary>
+/// Provides a set of extension methods for interacting with ANSI console
+/// functionality within the Kubernetes hosting context.
+/// This static class facilitates additional behaviors or utility methods
+/// that extend the base ANSI console capabilities.
+/// </summary>
+internal static partial class DockerComposePublisherLoggerExtensions
```

Comment:
What do you think @davidfowl time for us to start adopting this across Aspire?

Reply:
Ambivalent. Too much effort for little benefit at the moment. Can be done as cleanup on a class by class basis.